.github/workflows/graalwasm-micronaut-excelice.yml

```
@@ -0,0 +1,48 @@
+name: Test GraalWasm Micronaut Excelice Demo
```

Typo in the name. Please also rename the yml file.
```suggestion
name: Test GraalWasm Micronaut Excelize Demo
```